src/dbnode/storage/database_test.go

```diff
@@ -614,11 +614,13 @@ func TestDatabaseUpdateNamespace(t *testing.T) {
 	nses := d.Namespaces()
 	require.Len(t, nses, 2)
 
+	sr, err := namespace.LoadSchemaRegistry(namespace.GenTestSchemaOptions("namespace/schematest"))
+
```

Comment:
Can we add a check to make sure that the initial schema is not the same as the one that gets set?

Reply:
bump

Reply:
we did in database.go line292 method `namespaceDeltaWithLock`

this file is a test file, not database.go anymore.